docs/get-started/installation/from-source.md

```diff
@@ -56,6 +56,56 @@ cargo run node
 
 For more `witnet-rust` commands (`cli`, `wallet`, etc.) you can read the [Witnet-rust CLI documentation][CLI].
 
+## Run as `systemd` service
+
+* Create the user `witnet` dedicated to running the node:
+
+```
+sudo adduser witnet
+```
+
+* Install latest version with the user `witnet` dedicated to running the node:
+
+```
+sudo -u witnet ./systemd/runner.sh
+```
+
+* Copy the file `witnet.service` in `/lib/systemd/system/`:
+
+```
+sudo cp systemd/witnet.service /lib/systemd/system/witnet.service
+```
+
+* Enable service:
+
+```
+systemctl enable witnet.service
+```
+
+* Start service:
+
+```
+systemctl start witnet.service
+```
+
+* If for some reason the `witnet.service` file change, reload daemon:
+
+```
+systemctl daemon-reload
+```
+
+* Restart service:
+
+```
+systemctl restart witnet.service
+```
+
+* See the logs of service:
```

Comment:
```suggestion
* See the logs of the service:
```

Reply:
outdated, already added in the new version